followthemoney/mapping/property.py

```diff
@@ -133,5 +135,18 @@ def map(
                 splote.extend(value.split(self.split))
             values = splote
 
+        discarded_values: List[str] = []
+
         for value in values:
-            proxy.unsafe_add(self.prop, value, fuzzy=self.fuzzy, format=self.format)
+            added_value = proxy.unsafe_add(
+                prop=self.prop,
+                value=value,
+                cleaned=(not self.clean),
+                fuzzy=self.fuzzy,
+                format=self.format,
+            )
+
+            if value is not None and added_value is None:
+                discarded_values.append(value)
+
+        return discarded_values
```

Comment:
Again, I think it’s a little hacky to pass up the discarded values. Alternative might be to pass the `EntityMapping` to the `PropertyMapping` object for context, then do the logging directly in the `PropertyMapping.map` method.